broker/broker.go

```diff
@@ -60,22 +57,25 @@ func (b *CdnServiceBroker) Provision(
 		return spec, errors.New("must be invoked with `domain` and `origin` keys")
 	}
 
-	_, err = models.NewRoute(b.Settings, b.DB, instanceId, options.Domain, options.Origin)
+	_, err = b.Manager.Create(instanceId, options.Domain, options.Origin)
```

Comment:
Here's where the short receiver names come from: https://github.com/golang/go/wiki/CodeReviewComments#receiver-names. Go seems to favor short variable names in general, and very short receiver names in particular.